app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const { getTopics } = require("./controllers/controllers");
+const app = express();
+require('dotenv').config();
+
+
+app.use(express.json());
```

Comment:
This line isn't required until we do a post. Careful not to include logic extra to the ticket at hand